go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/ipfs/go-datastore v0.5.1
```

Comment:
For the changes in this file please see the commit message of https://github.com/keep-network/keep-core/pull/3291/commits/12a469082076cb9ae84441099e8bf370669211c8